### PR TITLE
[rtwcli] add rtw workspace delete

### PR DESCRIPTION
**Pull request by muritane**

### CODE REVIEW

File: rtwcli/rtw_cmds/rtw_cmds/workspace/verbs.py [hunk 1]
```diff
@@ -337,3 +396,138 @@ def main(self, *, args):
...
+    def delete_docker_image(self, docker_tag: str) -> None:
+        try:
+            subprocess.check_output(["docker", "rmi", docker_tag])
+            print(f"Docker image '{docker_tag}' has been deleted.")
+        except subprocess.CalledProcessError as e:
+            print(
+                f"Failed to delete docker image '{docker_tag}'. Error:\n{e.output.decode('utf-8')}"
+            )
+
+    def handle_delete_workspaces(
```
Comment (destogl) [inline rtwcli/rtw_cmds/rtw_cmds/workspace/verbs.py:456]: Can we ask if we should delete "src" folder please?

**Review by destogl — COMMENTED** [review]
I have tried to delete a workspace, but it didn't work. Is this inteded?

File: rtwcli/rtw_cmds/rtw_cmds/workspace/verbs.py [hunk 2]
```diff
@@ -337,3 +396,138 @@ def main(self, *, args):
...
+        ).ask()
+        if not confirm_delete:
+            return
+
+        ws_names_to_delete = [
+            choice_data[ws_choice_to_delete] for ws_choice_to_delete in ws_choices_to_delete
+        ]
+        self.handle_delete_workspaces(ws_names_to_delete, workspaces_config)
+
+    def delete_docker_image(self, docker_tag: str) -> None:
```
Comment (destogl) [inline rtwcli/rtw_cmds/rtw_cmds/workspace/verbs.py:447]: We should also delete docker containers related to workspace.

**Comment by muritane** [comment]
this branch is outdated

**Pull request closed without merging**